### ISSUE TITLE
GNOME 47 Support

### ISSUE THREAD

**Issue by aleb2000** [monyxie/inhibitionindicator#2]
Please update and release for GNOME 47.

**Comment by scarlion1** [comment]
In the mean time, manual update of the Metadata works for this.
Make a backup copy of `~/.local/share/gnome-shell/extensions/inhibitionindicator@monyxie.github.io/metadata.json` then open it in your favourite text editor and add a comma and `47` to `shell-version`, so it looks like this:
```
  "shell-version": [
    "45",
    "46",
    "47"
  ],
```
Save.  Then try Alt+F2 and `r` to restart the shell.  Extension should load.  If it doesn't, may need to log out of Gnome and log back in.  If it still doesn't load, you messed something up!  Check journal and system logs to see why, etc.

**Comment by develop7** [comment]
Yep, adding "47" to the manifest does it indeed. @monyxie could you do the magic?

**Comment by develop7** [comment]
Adding "48" there does the job as well. Could you @monyxie do the thing please?

**Comment by monyxie** [comment]
Version 5 with support for Gnome 47 and 48 has been uploaded to extensions.gnome.org.
